app/app.js

```diff
@@ -43,8 +47,8 @@ const server = app.listen(port, () => {
 // Index page - render the component list template
 app.get('/', function (req, res) {
   Promise.all([
-    dto.directoryToObject(path.resolve('./src/components')),
-    dto.directoryToObject(path.resolve('./src/views/examples'))
+    directoryToObject.directoryToObject(path.resolve(configPaths.components)),
```

Comment:
If we change

```
exports.directoryToObject = directoryToObject
```

to

```
exports = directoryToObject
```

in `lib/dto-helper.js` then I _think_ we can just make this `directoryToObject(path.resolve(configPaths.components))`

?

Reply:
mm..sounds good, but it has to be `module.exports`. updated.